Appends -norunpath to address publish error.
Will not send upstream.

--- ltmain.sh	2016-11-17 10:54:17.548942670 +0000
+++ ltmain.sh	2016-11-17 10:58:10.937162981 +0000
@@ -7332,7 +7332,8 @@
 	save_ifs="$IFS"; IFS='~'
 	for cmd in $cmds; do
 	  IFS="$save_ifs"
-	  eval cmd=\"$cmd\"
+	    # Append the command to remove runpath for CC.
+	  eval cmd=\"$cmd -norunpath\"
 	  $opt_silent || {
 	    func_quote_for_expand "$cmd"
 	    eval "func_echo $func_quote_for_expand_result"
@@ -7755,7 +7756,7 @@
 	func_warning "\`$output' will be relinked during installation"
       else
 	if test "$fast_install" != no; then
-	  link_command="$finalize_var$compile_command$finalize_rpath"
+	  link_command="$finalize_var$compile_command -norunpath"
 	  if test "$fast_install" = yes; then
 	    relink_command=`$ECHO "X$compile_var$compile_command$compile_rpath" | $Xsed -e 's%@OUTPUT@%\$progdir/\$file%g'`
 	  else
